packages/terra-form-select/src/tag/Frame.jsx

```diff
@@ -187,7 +187,7 @@ class Frame extends React.Component {
     this.input = input;
   }
 
-  getDisplay(displayId, ariaDescribedBy) {
+  getDisplay(displayId, ariaDescribedBy, id) {
```

Comment:
The id can probably be grabbed from the props so it doesn't need to be passed around as a parameter

Reply:
Added id as prop in 5bb136d